x/marker/keeper/keeper_test.go

```diff
@@ -1488,3 +1492,91 @@ func TestUpdateSendDenyList(t *testing.T) {
 		})
 	}
 }
+
+func TestReqAttrBypassAddrs(t *testing.T) {
+	// Tests both GetReqAttrBypassAddrs and IsReqAttrBypassAddr.
+	expectedNames := []string{
+		authtypes.FeeCollectorName,
+		rewardtypes.ModuleName,
+		quarantine.ModuleName,
+		govtypes.ModuleName,
+		distrtypes.ModuleName,
+		stakingtypes.BondedPoolName,
+		stakingtypes.NotBondedPoolName,
+	}
+
+	incByte := func(b byte) byte {
+		if b == 0xFF {
+			return 0x00
+		}
+		return b + 1
+	}
+
+	app := simapp.Setup(t)
+
+	for _, name := range expectedNames {
+		t.Run(fmt.Sprintf("get: contains %s", name), func(t *testing.T) {
+			expAddr := authtypes.NewModuleAddress(name)
+			actual := app.MarkerKeeper.GetReqAttrBypassAddrs()
+			assert.Contains(t, actual, expAddr, "GetReqAttrBypassAddrs()")
+		})
+	}
+	t.Run("get: only has expected entries", func(t *testing.T) {
+		// This assumes each expectedNames test passed. This is designed to fail if a new entry
+		// is added to the list (in app.go). When that happens, update the expectedNames with
+		// the new entry so it's harder for it to accidentally go missing.
+		actual := app.MarkerKeeper.GetReqAttrBypassAddrs()
+		assert.Len(t, actual, len(expectedNames), "GetReqAttrBypassAddrs()")
+	})
+
+	t.Run("get: called twice equal but not same", func(t *testing.T) {
```

Comment:
This seems like a pretty odd test. In what situations would the endpoint return two different sets of addresses when called back to back?

Reply:
This uses "equal" and "same" in relation to the `assert.Equal` and `assert.Same` tests.

It's asserting that `GetReqAttrBypassAddrs()` isn't just returning `k.reqAttrBypassAddrs` directly, but is instead, returning a deep copy of that slice. Each invocation should return the same content (be `.Equal`), but should be at different memory locations (`.NotSame`).